src/v/cluster/controller_backend.cc

```diff
@@ -101,7 +103,13 @@ void controller_backend::start_topics_reconciliation_loop() {
 
 void controller_backend::housekeeping() {
     if (!_topic_deltas.empty() && _topics_sem.available_units() > 0) {
-        (void)ss::with_gate(_gate, [this] { return reconcile_topics(); });
+        (void)ss::with_gate(_gate, [this] {
+            return reconcile_topics();
+        }).handle_exception([](const std::exception_ptr& e) {
```

Comment:
ah! we use `(void)ss::with_gate` so much i forget that it even returns a future.